packages/components/app/styles/components/alert.scss

```diff
@@ -0,0 +1,263 @@
+//
+// ALERT
+//
+// properties within each class are sorted alphabetically
+//
+//
+
+@use "../mixins/focus-ring" as *;
+
+.hds-alert {
+  align-items: flex-start;
+  display: flex;
+}
+
+// ICON
+
+.hds-alert__icon {
+  flex: none;
+  height: 20px;
+  margin-right: 12px;
+  width: 20px;
+
+  .hds-alert--type-compact & {
+    height: 14px;
+    width: 14px;
+  }
+}
+
+// CONTENT (TEXT + ACTIONS + GENERIC)
+
+.hds-alert__content {
+  flex: 1 1 auto;
+}
+
+// TEXT (TITLE & DESCRIPTION)
+
+.hds-alert__text {
+  color: var(--token-color-foreground-warning-on-surface);
+  display: flex;
+  flex-direction: column;
+  font-family: var(--token-typography-body-200-font-family);
+  font-size: var(--token-typography-body-200-font-size);
+  justify-content: center;
+  line-height: var(--token-typography-body-200-line-height);
+
+  .hds-alert--type-compact & {
+    font-family: var(--token-typography-body-100-font-family);
+    font-size: var(--token-typography-body-100-font-size);
+    line-height: var(--token-typography-body-100-line-height);
+  }
+}
+
+.hds-alert__title {
+  font-weight: var(--token-typography-font-weight-semibold);
+}
+
+.hds-alert__description {
+  color: var(--token-color-foreground-primary);
+  font-weight: var(--token-typography-font-weight-regular);
+
+  .hds-alert__title + & {
+    margin-top: 4px;
+  }
+
+  // we add very basic styling for elements that may be injected via the "description" string
+
+  strong {
+    font-weight: var(--token-typography-font-weight-medium);
+  }
+
+  code, pre {
+    font-family: var(--token-typography-font-stack-code);
```

Comment:
I didn't use the `code-100-font-size/line-height` because they would be `13px/18px` which is smaller than the styles used for the `description` text (`body-200-font-size/line-height` = `18px/20px`) so I just needed the `font-family` in this case.

Reply:
Ok, let's leave it as is for now. We should have a separate discussion to discuss our code style in general because it seems like the one we have is intended to be used as a code block and not within body text, so we may need to decide whether we want to add another style and tokens or not.

Reply:
> We should have a separate discussion to discuss our code style in general because it seems like the one we have is intended to be used as a code block and not within body text, so we may need to decide whether we want to add another style and tokens or not.

@cveigt can you please set up a meeting for next week, me you and @heatherlarsen so we can align on what steps to take for this? Thanks

Reply:
Just as an FYI, the need for additional code styles has already come up as DevDot relies on them heavily. There is an [Asana task](https://app.asana.com/0/1202008719910219/1202214539819773) (internal) for this work created in the backlog.